src/ping.c

```diff
@@ -462,15 +465,33 @@ static int ping_config(const char *key, const char *value) /* {{{ */
       return 1;
     }
 
+    if ((value_alias = strchr(value, ' ')) == NULL)
```

Comment:
For doing non-trivial configurations, the simple config callback is not the right choice anymore. Instead, this should use the "complex" config callback to get the two values individually.